src/main/holochainManager.ts

```diff
@@ -340,23 +339,20 @@ export class HolochainManager {
    * @param appId
    * @param distributionInfo
    * @param networkSeed
-   * @param membrane_proofs
    * @param icon
    */
   async installWebHappFromBytes({
     happAndUiBytes,
     appId,
     distributionInfo,
     networkSeed,
-    membrane_proofs,
     icon,
     agentPubKey,
   }: {
     happAndUiBytes: HappAndUiBytes;
     appId: string;
     distributionInfo: DistributionInfoV1;
     networkSeed?: string;
-    membrane_proofs?: { [key: string]: MembraneProof };
```

Comment:
This membrane_proof argument was here as optional parameter but unused because there is no associated UI to provide memproofs at install time.